examples/managed_queue/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "managed_queue"
+version = "0.0.1"
```

Comment:
Can you set this to 0.0.0 to make sure it doesn't accidentally get published to `crates.io`?